server/@types/bapv.d.ts

```diff
@@ -106,10 +106,12 @@ export type VisitSessionData = {
   visitSlot?: VisitSlot
   originalVisitSlot?: VisitSlot
   visitRestriction?: 'OPEN' | 'CLOSED'
+  visitorIds?: number[]
```

Comment:
Add `visitorIds` to session so don't need to rely on having `visitors` populated. This means that at the start of an update journey, just the IDs can be used to pre-select existing visitors.